src/app/view/components/_common/push-publish-dialog/push-publish-dialog.component.html

```diff
@@ -0,0 +1,51 @@
+<p-dialog
+    modal="modal"
+    width="500"
+    [dismissableMask]="true"
+    [(visible)]="show"
+    [responsive]="true"
+    [resizable]="false"
+    [draggable]="false"> 
+
+    <p-header>
+        {{dotMessageService.get('contenttypes.content.push_publish')}}
+    </p-header>
+
+    <form [formGroup]="form" (ngSubmit)="submitPushAction($event)" novalidate>
```

Comment:
This forms needs the required message component: https://github.com/dotCMS/core-web/blob/master/src/app/portlets/content-types/form/content-types-form.component.html#L12

And also all the form unless we said no.